integration/spark-common-cluster-test/src/test/scala/org/apache/carbondata/cluster/sdv/generated/CarbonV1toV3CompatabilityTestCase.scala

```diff
@@ -49,8 +49,10 @@ class CarbonV1toV3CompatabilityTestCase extends QueryTest with BeforeAndAfterAll
       .getOrCreateCarbonSession(storeLocation, metaLocation).asInstanceOf[CarbonSession]
     println("store path from env : " + CarbonEnv.getInstance(localspark).storePath)
     localspark.sparkContext.setLogLevel("WARN")
-    localspark.sessionState.asInstanceOf[CarbonSessionState].metadataHive
-      .runSqlHive(
+    localspark.asInstanceOf[CarbonSession].asInstanceOf[CarbonSession].sharedState.externalCatalog
+      .asInstanceOf[HiveExternalCatalog].client.runSqlHive(
```

Comment:
It is not required to cast twice

Reply:
I just commented this line temporarily, use spark.run to run this sql, because class HiveExternalCatalog cannot be accessed in non-org.apache.spark package.